src/styles/common.scss

```diff
@@ -90,6 +90,24 @@
     color: var(--primary1);
 }
 
+.transparentButton {
+    border: none;
+    background: transparent;
+    color: inherit;
+}
+.button:not(:disabled):hover {
+    background: transparent;
+    color: inherit;
+}
```

Comment:
There's already a different rule for `.button:not(:disabled):hover` above in this file. And it has different styling.

Reply:
Good catch. It was meant to be `.transparentButton:not(:disabled):hover`